projects/framework-blender/extensions/tool-configs/blender-scene-publisher.yaml

```diff
@@ -0,0 +1,108 @@
+type: tool_config
+name: blender-scene-publisher
+config_type: publisher
+engine:
+  - type: plugin
+    tags:
+      - context
+    plugin: store_asset_context
+    options:
+      asset_type_name: scene
+    ui: publisher_asset_version_selector
+
+  # Export the snapshot component.
+  - type: group
+    tags:
+      - component
+    options:
+      component: snapshot
+    #optional: False by default so is no need
+    #enabled: True by default so is no need
+    plugins:
+      - type: plugin
+        plugin: store_component
+      - type: plugin
+        tags:
+          - collector
+        plugin: blender_scene_collector
+        ui: blender_scene_options_selector
+        options:
+          export_type: scene # selection is also available
+          extension_format: blend
```

Comment:
Does this have other options?

Reply:
not for now, but keeping it open for later.

Reply:
I think extension_format should be removed, and export_type should be kept and align plugins to be able to export selection. Otherwise, remove it as well.